prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/WA/WA-receiver-transform.yml

```diff
@@ -0,0 +1,155 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+  FHS: '/FHS'
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+  ORC: '/PATIENT_RESULT/ORDER_OBSERVATION/ORC'
+  OBR: '/PATIENT_RESULT/ORDER_OBSERVATION/OBR'
+
+elements:
+  - name: wa-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: wa-file-sending-application-namespace-id
+    condition: 'FHS.exists()'
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{FHS}-3-1' ]
+
+  - name: wa-file-sending-application-universal-id
+    condition: 'FHS.exists()'
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{FHS}-3-2' ]
+
+  - name: wa-file-sending-application-universal-id-type
+    condition: 'FHS.exists()'
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{FHS}-3-3' ]
+
+  - name: wa-sending-application-namespace-id
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{MSH}-3-1' ]
+
+  - name: wa-sending-application-universal-id
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{MSH}-3-2' ]
+
+  - name: wa-sending-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-3-3' ]
+
+  - name: wa-receiving-application-namespace-id
+    value: [ '"WADOHPHRED"' ]
+    hl7Spec: [ '%{MSH}-5-1' ]
+
+  - name: wa-receiving-application-universal-id
+    value: [ '"2.16.840.1.113883.3.237.4.2"' ]
+    hl7Spec: [ '%{MSH}-5-2' ]
+
+  - name: wa-receiving-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-5-3' ]
+
+  - name: wa-receiving-facility-namespace-id
+    value: [ '"dn1fro00"' ]
+    hl7Spec: [ '%{MSH}-6-1' ]
+
+  - name: wa-receiving-facility-universal-id
+    value: [ '"1.3.6.1.4.1.38630.2.1.1.19"' ]
+    hl7Spec: [ '%{MSH}-6-2' ]
+
+  - name: wa-receiving-facility-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-6-3' ]
+
+  - name: wa-race-identifier
```

Comment:
Where in the setting that tell us to replace all these PIDs?

Reply:
I removed them so they were not clogging up the settings even more but this was their replaceValueAwithB value before:
        FHS-3:
        - '*': "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO"
        FHS-5:
        - '*': "WADOHPHRED^2.16.840.1.113883.3.237.4.2^ISO"
        FHS-6:
        - '*': "dn1fro00^1.3.6.1.4.1.38630.2.1.1.19^ISO"
        MSH-3:
        - '*': "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO"
        MSH-5:
        - '*': "WADOHPHRED^2.16.840.1.113883.3.237.4.2^ISO"
        MSH-6:
        - '*': "dn1fro00^1.3.6.1.4.1.38630.2.1.1.19^ISO"
        PID-10:
        - ? ""
          : "UNK^unknown^NULLFL^^^^20090501"
        PID-15:
        - ? ""
          : "333^Unknown^99WA_LANG^^^^1"
        PID-22:
        - ? ""
          : "U^Unknown^HL70189^^^^HL7v2.5.1"